### DISCUSSION TITLE
Sponsoring Flutterfire Contributor to add Windows Support

### DISCUSSION THREAD

**Discussion by markbreuss** [firebase/flutterfire#6876] [Feature request] [answered]
Good Day,

we are an Austrian based software company which uses Flutter and Firebase.
Since we had huge pressure to add Windows Support we currently use: https://github.com/cachapa/firedart.
However there are many important features missing and we need a production ready implementation ASAP.

Thus we would like to sponsor a contributor to start windows support for FlutterFire.

If this sounds interesting contact me at mark.breuss@markone.at

Assuming more people face similar problems, a joint venture could allow a contributor to spend more time on the issue.
https://github.com/FirebaseExtended/flutterfire/discussions/6071
https://github.com/FirebaseExtended/flutterfire/discussions/6741
https://github.com/FirebaseExtended/flutterfire/discussions/6233
https://github.com/FirebaseExtended/flutterfire/discussions/6071
https://github.com/FirebaseExtended/flutterfire/discussions/5557

Greetings,

Mark

**Comment by Salakar** [comment, marked as answer]
Closing in favour of #5557

Note: this is now planned, more on this soon.
